pkg/containers/runtime/crio.go

```diff
@@ -55,6 +55,7 @@ func (e *crioEnricher) Get(containerId string, ctx context.Context) (ContainerMe
 	}
 	metadata.Name = resp.Status.Metadata.Name
 	metadata.Image = resp.Status.Image.Image
+	metadata.ImageDigest = strings.TrimPrefix(resp.Status.ImageRef, "sha256:")
```

Comment:
why do you need to trim the prefix? its part of the digest

Reply:
Fixed.